citrination_client/models/data_view.py

```diff
@@ -1,6 +1,6 @@
 class DataView(object):
 
-    def __init__(self, view_id, name, description, column_names, columns=[]):
+    def __init__(self, view_id, name, description, datasets, column_names=None, columns=[]):
```

Comment:
I'd give `datasets` a default value of [] and column_names a default value of [].